Meta/Azure/Lagom.yml

```diff
@@ -158,6 +158,16 @@ jobs:
           ASAN_OPTIONS: 'strict_string_checks=1:check_initialization_order=1:strict_init_order=1:detect_stack_use_after_return=1'
           UBSAN_OPTIONS: 'print_stacktrace=1:print_summary=1:halt_on_error=1'
           TESTS_ONLY: 1
+      # Publish test dumps as an artifact.
```

Comment:
Minor CI YML legibilitly comment: newlines between steps please :smile: (so, here, and before `- publish`).

Reply:
:+1: